react/src/Analyses/index.tsx

```diff
@@ -257,6 +294,7 @@ export default function Analyses() {
                 title: "Path Prefix",
                 field: "result_path",
                 type: "string",
+                render: rowData => <TruncatedText text={rowData.result_path} maxLength={15} />,
```

Comment:
We have a `Note` component that also provides a similar functionality to this which we use for the Notes column.